src/main/java/com/cleanroommc/groovyscript/compat/mods/prodigytech/ExplosionFurnaceAdditives.java

```diff
@@ -0,0 +1,115 @@
+package com.cleanroommc.groovyscript.compat.mods.prodigytech;
+
+import com.cleanroommc.groovyscript.api.IIngredient;
+import com.cleanroommc.groovyscript.api.documentation.annotations.Example;
+import com.cleanroommc.groovyscript.api.documentation.annotations.MethodDescription;
+import com.cleanroommc.groovyscript.api.documentation.annotations.RegistryDescription;
+import com.cleanroommc.groovyscript.helper.ingredient.OreDictIngredient;
+import com.cleanroommc.groovyscript.registry.VirtualizedRegistry;
+import lykrast.prodigytech.common.recipe.ExplosionFurnaceManager;
+import net.minecraft.item.ItemStack;
+
+class EFAdditiveRecipe {

```

Comment:
could you move this to a private inner class

Reply:
would have to be public because
`'com.cleanroommc.groovyscript.compat.mods.prodigytech.ExplosionFurnaceAdditives.EFAdditiveRecipe' has private access in 'com.cleanroommc.groovyscript.compat.mods.prodigytech.ExplosionFurnaceAdditives'`
but otherwise yes I'll do it